wsmaster/che-core-api-workspace/src/test/resources/devfile/schema_test/devfile/devfile_v2-1-0-alpha_just_schemaVersion.yaml

```diff
@@ -11,4 +11,4 @@
 #
 
 ---
-schemaVersion: 2.1.0-alpha
+schemaVersion: 2.1.0
```

Comment:
file is still named 2-1-0-alpha

Reply:
sorry for missing that. BTW we'll discuss on Monday if we really should use -alpha prefix, so I hope we can avoid adapting devfiles after release.